apps/dapp/src/hooks/use-coreV2-strategy-data.tsx

```diff
@@ -0,0 +1,80 @@
+import environmentConfig from 'constants/env';
+import { useState } from 'react';
+import useInterval from 'use-interval';
+import { fetchGenericSubgraph } from 'utils/subgraph';
+
+export interface V2StrategyDailySnapshot {
+    timestamp: string;
+    timeframe: string;
+    strategy: { name: string };
+    totalMarketValueUSD: string;
+    debtUSD: string;
+    creditUSD: string;
+    principalUSD: string;
+    accruedInterestUSD: string;
+    nominalEquityUSD: string;
+    nominalPerformance: string;
+    benchmarkedEquityUSD: string;
+    benchmarkPerformance: string;
+}
+
+export type V2StrategyMetric = keyof Omit<V2StrategyDailySnapshot, 'timeframe' | 'timestamp' | 'strategy'>;
+
+
+async function fetchCoreV2<RawResponse extends Record<string, unknown>>(query: string) {
+    const resp: RawResponse = (await fetchGenericSubgraph(environmentConfig.subgraph.templeV2, query)).data
+    return resp
+}
+
+
+async function fetchStrategyDailySnapshots(now: Date) {
+    // TODO: filter strategies here?
+    let since = Math.floor((now.getTime() - 365 * 24 * 60 * 60 * 1000) / 1000).toString()
```

Comment:
Nit: Put the value here as a named constant, for example `const ONE_YEAR_MS = 365*24....`

Just makes it easier to read.